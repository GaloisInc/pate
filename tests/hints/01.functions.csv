thunk_FUN_code_00005d, code:000000
FUN_code_00005d, code:00005d
FUN_code_0000d6, code:0000d6
__do_clear_bss, code:0000dc
