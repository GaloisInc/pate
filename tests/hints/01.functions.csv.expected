VerificationHints { blockMappings = []
                  , functionEntries = [ ("thunk_FUN_code_00005d", FunctionDescriptor { functionSymbol = "thunk_FUN_code_00005d", functionAddress = 0x0, functionArguments = [] })
                                      , ("FUN_code_00005d", FunctionDescriptor { functionSymbol = "FUN_code_00005d", functionAddress = 0x5d, functionArguments = [] })
                                      , ("FUN_code_0000d6", FunctionDescriptor { functionSymbol = "FUN_code_0000d6", functionAddress = 0xd6, functionArguments = [] })
                                      , ("__do_clear_bss", FunctionDescriptor { functionSymbol = "__do_clear_bss", functionAddress = 0xdc, functionArguments = [] })
                                      ]
                  , dataSymbols = []
                  }
